Library/Homebrew/test/test_caveats.rb

```diff
@@ -18,7 +18,8 @@ def test_empty?
     f = formula do
       url "foo-1.0"
 
-      def caveats
+      # let's disable rubocop here since it dislikes nested method definitions
+      def caveats # rubocop:disable NestedMethodDefinition
```

Comment:
Instead of all these, add a `.rubocop.yml` file in `Library/Homebrew/test` with:

```
inherit_from: ../../.rubocop.yml

# `formula do` needs uses method definitions
Style/NestedMethodDefinition:
  Enabled: false
```